Source/Events/PartitionId.cs

```diff
@@ -9,25 +9,25 @@ namespace Dolittle.SDK.Events
     /// <summary>
     /// Represents the concept of a unique identifier for a partition.
     /// </summary>
-    public class PartitionId : ConceptAs<Guid>
+    public class PartitionId : ConceptAs<string>
     {
         /// <summary>
         /// The <see cref="PartitionId"/> that refers to no partition.
         /// </summary>
-        public static readonly PartitionId Unspecified = Guid.Empty;
+        public static readonly PartitionId Unspecified = string.Empty;
```

Comment:
I was unsure of this when I worked on the JavaScript part of this. I think what I did is that I set Unspecified to be the empty guid, as it was before. What do we actually want? We may have some documentation that documents that